catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/ResourceOperations.java

```diff
@@ -731,6 +688,7 @@ public URI getResourceUri() {
      *
      * @param metacard the {@link Metacard} to get the supported options for
      * @return the {@link Set} of supported options for the metacard
+     * @deprecated
```

Comment:
why are these now deprecated? If there is an alternate / preferred method to use we should refer to that.